taskcluster/fenix_taskgraph/transforms/build.py

```diff
@@ -35,14 +35,15 @@ def add_shippable_secrets(config, tasks):
 
         if task.pop("include-shippable-secrets", False) and config.params["level"] == "3":
             build_type = task["attributes"]["build-type"]
+            gradle_build_type = task["run"]["gradle-build-type"]
```

Comment:
This value is defined in the `kind.yml`, like at that place https://github.com/mozilla-mobile/fenix/blob/35543ec6b9b84ab47d22dc7e0c4c69bc725a859a/taskcluster/ci/build/kind.yml#L62